app/templates/main/browse-transferring-body.html

```diff
@@ -16,20 +16,22 @@
 {% endwith %}
 <form action="{{ url_for('main.browse_transferring_body', _id=request.view_args['_id']) }}"
       method="get">
-    {% if view_type == "desktop" %}
+    <div class="browse-layout">
         <!-- SORT -->
         <div class="govuk-form-group sort-container__form">
             {% with dict = sorting_list %}
                 {% include "sorting-list.html" %}
             {% endwith %}
         </div>
         <!-- TABLE -->
-        <div class="govuk-grid-column-two-thirds browse-grid--two-thirds">
+        <div class="govuk-grid-column-two-thirds browse-grid--two-thirds mobile-table">
```

Comment:
would it make more sense to rename mobile-table to something different since its a bit misleading when it is applied for both desktop and mobile

Reply:
I was initially thinking that creating a class for this type of table, where we overide the mobile properties we care about for the mobile witdth. 

But, this is fine, its like a mixin

Reply:
Removed mobile-filters & now using browse-grid--mobile-table to keep in sync with other class names

Reply:
Ah gotcha, I think I've done that for the filters panel and table now